doc/modchannels.adoc

```diff
@@ -0,0 +1,90 @@
+= Modchannels
+
+Allows server side mods (SSMs) communication to client side mods (CSMs) and vice versa.
+
+== Server Side API
+
+=== Functions
+
+==== minetest.mod_channel_join(channel_name)
+
+* `channel_name`: string, modchannel name
+
+Returns a object for use with <<Methods>>. Creates the channel if it does not exist and joins the channel.
+
+==== minetest.register_on_modchannel_message(function(channel_name, sender, message))
+
+* `channel_name`: string, modchannel name (already joined)
+* `sender`: string, empty if from a ssm, player name if from a client
+* `message`: string, message
```

Comment:
This needs an explanation for what it does/is useful for.

Reply:
https://github.com/minetest/minetest_docs/pull/15/commits/3940703fc9579f4c929b80eb614a62b11b20dbbf